src/plugins/wizard/public/application/components/top_nav.tsx

```diff
@@ -5,14 +5,29 @@
 
 import React, { useMemo } from 'react';
 import { useParams } from 'react-router-dom';
+import useUnmount from 'react-use/lib/useUnmount';
```

Comment:
nit: Use this instead

```
import { useUnmount } from 'react-use';
```

Always use named exports from the root of the library when possible since the underlying structure may change in future